### PR TITLE
[SELF INCOMPATIBLE] xcmd: remove xcmd feature

### PR DESCRIPTION
**Pull request by masatake**
I'm very sad that I have to make this pull request.

I will remove XCMD from the planed first stable release(6.0).

I like the idea but it makes difficult to keep the consistency of ctags internal.
I guess only a few people using the feature.

Universal-ctags manages kinds more strictly than Exuberant-ctags.
Many of tags generators(xcmds) for specific languages models Exuberant-ctags.
The current Universal-ctags cannot fill the gap in kinds managment.

One of the way to go is that Universal-ctags defines better protocol between
Universal-ctags and xcmds that fill the gaps. However, to define such
protocol, I need much more experiences.

#705 (Process Multiple files with xcmd ) porints the critical weakness of
XCMD feature.

Please, give us feed-back via https://github.com/universal-ctags/ctags/issues/1442

Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

File: win32/ctags_vs2013.vcxproj.filters [hunk 1]
```diff
@@ -621,4 +615,4 @@
       <Filter>Header Files</Filter>
     </ClInclude>
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```
Comment (masatake) [inline win32/ctags_vs2013.vcxproj.filters]: I should not do this.

**Comment by masatake** [comment]
Bye, xcmd.

**Comment by k-takata** [comment]
`libexec/drivers/coffeetags` still exists. Shouldn't it removed?

**Comment by masatake** [comment]
Yes, this should be removed. Can I ask you to open an issue about it?
I'm afraid that I cannot remember this.

**Pull request merged**